ktor-server/ktor-server-plugins/ktor-server-call-id/jvm/src/io/ktor/server/plugins/CallId.kt

```diff
@@ -40,207 +38,194 @@ public class RejectedCallIdException(
 }
 
 /**
- * A call id that is retrieved or generated by [CallId] plugin or `null` (this is possible if there is no
- * call id provided and no generators configured or [CallId] plugin is not installed)
+ * [CallId] plugin's configuration
  */
-public val ApplicationCall.callId: String? get() = attributes.getOrNull(CallId.callIdKey)
+public class CallIdConfiguration {
```

Comment:
as from https://github.com/ktorio/ktor/pull/2782#discussion_r787789014:
> We agree to rename Configuration to Config everywhere and put prefixes with a clear name.

Reply:
Thanks for pointing this out, changed to `CallIdConfig`